src/abstractions/configuration.ts

```diff
@@ -9,4 +9,12 @@ export type AppConfiguration = {
   readonly htmlEditorApiBaseUrl: string;
   readonly dopplerRestApiBaseUrl: string;
   readonly useDummies: boolean;
+  readonly exitMenuItemsUrls: exitMenuItemsUrls;
+};
+
+export type exitMenuItemsUrls = {
+  readonly home: string;
+  readonly campaigns: string;
+  readonly lists: string;
+  readonly controlPanel: string;
 };
```

Comment:
I think that these URLs can be reusable, I mean `https://app2.fromdoppler.com/Lists/SubscribersList` is the URL of the lists page, not only the URL for the exit to list.

What do you think about some of these options:

```suggestion
  readonly homeUrl: string;
  readonly campaignsUrl: string;
  readonly listsUrl: string;
  readonly controlPanelUrl: string;
};

export type dopplerExternalUrls = {
  readonly home: string;
  readonly campaigns: string;
  readonly lists: string;
  readonly controlPanel: string;
}
```

or

```suggestion
  readonly dopplerExternalUrls: dopplerExternalUrls;
};

export type dopplerExternalUrls = {
  readonly home: string;
  readonly campaigns: string;
  readonly lists: string;
  readonly controlPanel: string;
}
```